integration/integrate-with-cloud-sources.adoc

```diff
@@ -0,0 +1,37 @@
+[id="integrate-with-cloud-sources"]
+= Integrating with cloud sources
+include::modules/common-attributes.adoc[]
+:context: integrate-with-cloud-sources
+
+toc::[]
+
+[role="_abstract"]
+You can integrate {product-title} with different cloud sources to discover potential clusters
+to secure. The purpose of the cluster discovery is to gain a detailed overview over the
+cluster assets which are already or not yet secured by {product-title-short}.
+The clusters that have been discovered from a cloud source are accessible via
+*Platform Configuration* -> *Clusters* -> *Discovered clusters*.
+
+Discovered clusters are matched against already secured clusters via metadata that is collected
```

Comment:
nit: not sure whether the Sensor metadata part is really important here, since we don't _really_ expose the metadata anywhere so it might be confusing.